algebird-core/src/main/scala/com/twitter/algebird/HyperLogLogSeries.scala

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2014 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.twitter.algebird
+
+case class HLLSeries(bits: Int, rows: Vector[Map[Int, Long]]) {
```

Comment:
We need more docs here. I have to read carefully to remind myself of the intent and use of this code. Can you give some comments that will show up in the scaladoc?